charts/kuberpult/values.yaml

```diff
@@ -343,3 +343,5 @@ manageArgoApplications:
   enabled: false
   # List of teams that should be self managed by the rollout service.
   filter: []
+
+generateMigrations: false
```

Comment:
I don't think this flag is necessary. We should just always generate the migrations.